packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -61,6 +62,8 @@ export const ChatList = ({
   const chatListButtonItems = props.buttonItems === undefined ? [] : props.buttonItems;
   const [menuItems, setMenuItems] = useState<ChatListMenuItem[]>(props.menuItems === undefined ? [] : props.menuItems);
   const [selectedItem, setSelectedItem] = useState<string>();
+  const [readItems, setReadItems] = useState<string[]>([]);
```

Comment:
It seems like we shouldn't have this at all. ChatList is the owner of what is read and not, but ChatListItems already have a state for whether things are read or not.